config/v1/types_cluster_operator.go

```diff
@@ -176,10 +176,15 @@ const (
 	// unexpected errors are handled as operators mature.
 	OperatorDegraded ClusterStatusConditionType = "Degraded"
 
-	// Upgradeable indicates whether the operator safe to upgrade based on the current cluster state. When status is `False`
-	// administrators should not upgrade their cluster and the message field should contain a human readable description
-	// of what the administrator should do to allow the operator to successfully update.  A missing condition, True,
-	// and Unknown are all treated by the CVO as allowing an upgrade.
+	// Upgradeable indicates whether the operator is safe to upgrade based on the
+	// current cluster state. When status is False, the cluster-version operator
+	// will prevent the cluster from performing impacted updates unless forced.
```

Comment:
what do you mean by "impacted updates"?

isn't it just "we allow z-stream updates and we don't allow minor updates"?  if so, let's just state that.

Reply:
no, the CVO will also block patch updates for ClusterVersion `spec.overrides`.  Discussion in [rhbz#1822844][1], linked from the topic post.

[1]: https://bugzilla.redhat.com/show_bug.cgi?id=1822844

Reply:
got it, thanks.  adding suggestion below.